react/features/stream-effects/blur/index.js

```diff
@@ -7,28 +7,31 @@ import createTFLiteModule from './vendor/tflite/tflite';
 import createTFLiteSIMDModule from './vendor/tflite/tflite-simd';
 
 const models = {
-    'model96': 'libs/segm_lite_v681.tflite',
-    'model144': 'libs/segm_full_v679.tflite'
+    model96: 'libs/segm_lite_v681.tflite',
+    model144: 'libs/segm_full_v679.tflite'
 };
 
 const segmentationDimensions = {
-    'model96': {
-        'height': 96,
-        'width': 160
+    model96: {
+        height: 96,
+        width: 160,
+        virtualBackground: {}
```

Comment:
This is not needed.

Reply:
If I remove this "virtualBackground: {}" -> on npm run lint I have this output: "Cannot assign virtualBackground to options.virtualBackground because: property virtualBackground is missing in object literal"

Reply:
Please create a new object then instead of augmenting this one. Like so:

```
    const options = {
       ...(wasmCheck.feature.simd ? segmentationDimensions.model144 : segmentationDimensions.model96),
       virtualBackground
   };

   return new JitsiStreamBackgroundEffect(tflite, options);
```